api/internal/localizer/builtinplugins.go

```diff
@@ -91,7 +91,7 @@ func (lbp *localizeBuiltinPlugins) Filter(plugins []*yaml.RNode) ([]*yaml.RNode,
 				},
 			},
 			yaml.FilterFunc(func(node *yaml.RNode) (*yaml.RNode, error) {
-				isHelm := node.GetApiVersion() == konfig.BuiltinPluginApiVersion &&
+				isHelm := (node.GetApiVersion() == konfig.BuiltinPluginApiGroup || node.GetApiVersion() == konfig.BuiltinPluginApiVersion) &&
```

Comment:
is this correct? comparing the version to the group?

Reply:
I'm little confuse that why we compare node.GetApiVersion with group and version. I think adding code comment here will help resolve it